demo-server.html

```diff
@@ -238,7 +236,7 @@
                 class="bold">OMERO.insight</span> client for your platform by
                 clicking on the appropriate link on the <a class="reference
                 external"
-                href="http://downloads.openmicroscopy.org/latest/omero5.3/"
+                href="https://downloads.openmicroscopy.org/latest/omero/"
```

Comment:
We probably want to replace all the downloads links with https://www.openmicroscopy.org/omero/downloads/ given that the downloads site is just folders of artifacts now rather than a rendered page.